pkgs/servers/photoprism/libtensorflow.nix

```
@@ -0,0 +1,89 @@
+{ lib, stdenv, fetchurl, ... }:
+let
+  inherit (stdenv.hostPlatform) system;
+in
+stdenv.mkDerivation rec {
+  pname = "libtensorflow-photoprism";
+  version = "1.15.2";
+
+  srcs = [
+    # Photoprism-packaged libtensorflow tarball (with pre-built libs for both arm64 and amd64)
+    # We need this specific version because of https://github.com/photoprism/photoprism/issues/222
+    (fetchurl {
+      sha256 = {
+        x86_64-linux = "sha256-bZAC3PJxqcjuGM4RcNtzYtkg3FD3SrO5beDsPoKenzc=";
+        aarch64-linux = "sha256-qnj4vhSWgrk8SIjzIH1/4waMxMsxMUvqdYZPaSaUJRk=";
+      }.${system} or (throw "Unsupported system");
+
+      url =
+        let
+          systemName = {
+            x86_64-linux = "amd64";
+            aarch64-linux = "arm64";
+          }.${system} or (throw "Unsupported system");
+        in
+        "https://dl.photoprism.app/tensorflow/${systemName}/libtensorflow-${systemName}-${version}.tar.gz";
```

Same here, it's up to your preference.
```suggestion
      url = let
        systemName = {
          x86_64-linux = "amd64";
          aarch64-linux = "arm64";
        }.${system};
      in "https://dl.photoprism.app/tensorflow/${systemName}/libtensorflow-${systemName}-${version}.tar.gz";
```

---

Looks good